src/python/crash_analysis/severity_analyzer.py

```diff
@@ -104,8 +104,20 @@ def get_security_severity(crash_type, crash_output, job_name,
 class SeverityAnalyzerSanitizer(object):
   """Generic ASan severity analyzer."""
 
-  def analyze(self, crash_type, _, requires_gestures):
+  def analyze(self, crash_type, crash_output, requires_gestures):
```

Comment:
@inferno-chromium what do you think of this?
Maybe it's better to have exceptions for SecurityExceptionCritical, SecurityExceptionHigh, SecurityExceptionMedium, SecurityExceptionLow rather than make this API more complex.

Reply:
For the target.
I mean that right now we are treating SecurityExceptionCritical, SecurityExceptionHigh etc all as `uncaught exception` crash type.
So to determine the severity of uncaught exception we need to pass crash_output to `analyze`.
But instead of doing that, when determining crash type, we can make the type `SecurityExceptionCritical` or `SecurityExceptionHigh` etc if needed.